boudicca.base/publisher-event-html/src/main/resources/static/js/main.js

```diff
@@ -72,19 +80,23 @@ document.addEventListener("DOMContentLoaded", () => {
 
   const onSearchLoadMoreButtonBehaviour = (response) => {
     const loadMoreButton = document.getElementById("loadMoreButton");
-    const endOfResultsInfo = document.getElementById("endOfResults");
-    if (!loadMoreButton || !endOfResultsInfo) {
+    // const endOfResultsInfo = document.getElementById("endOfResults");
+    const resultNotFoundLink = document.getElementById("resultNotFound");
+    const searchText = document.getElementById("searchText");
+    const searchInputKeyword = searchInput.value;
+    if (!loadMoreButton || !resultNotFoundLink) {
       return;
     }
 
-    if (!response) {
-      if (loadMoreButton) {
-        loadMoreButton.style.display = "none";
-        endOfResultsInfo.style.display = "block";
-      }
+    if (!response.trim()) {
+      searchText.textContent = `"${searchInputKeyword}"`;
```

Comment:
i am not quite sure if this is the best way to handle this. some concerns i have are
1) the search input can and probably often is empty, especially if the user just selects some filters.
2) the text is kinda confusing if you click the load more button and then you get nothing back

Reply:
it still is like that?